### PR TITLE
Recommend workstation backup as step in the install guide

### LINKED ISSUE #4265: [docs] Recommend workstation backup as step in the install guide
We have a [topic guide](https://docs.securedrop.org/en/release-0.12.0/backup_workstations.html) for backing up workstations. It even starts with the words "Now that you have set up [the workstations]", but as far as I can tell, we do not state it as a strongly recommended step in the installation guide itself.

Problems with corrupted or misconfigured Admin Workstations are quite common, so I suggest we strongly recommend making a backup during the install, to get admins into the habit of doing it regularly.

### PR DESCRIPTION
**Pull request by cji**
## Status

Ready for review

## Description of Changes

Fixes #4265.

Changes proposed in this pull request: recommend workstation backup as step in the install guide

## Testing

Check for grammar, placement of instructions in the guide.

## Deployment

Any special considerations for deployment? No.

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

**Comment by conorsch** [comment]
Thank you for the contribution, @cji ! Flagging that the `lint` failure in CI appears to be unrelated to your changes, likely caused by https://github.com/freedomofpress/securedrop/issues/4301. We'll work on that to get these tests passing and proceed with review.

**Comment by cji** [comment]
Thanks for the clarification @eloquence! That totally makes sense. I'll re-work this PR to address those changes.

**Comment by cji** [comment]
looks like some additional unrelated CI failures? At least, I don't think this PR should affect grsec kernel things 😅

**Comment by conorsch** [comment]
> looks like some additional unrelated CI failures? At least, I don't think this PR should affect grsec kernel things

Aye! Caused by #4308, specifically. FYI, it's possible to skip the full CI run if you prefix your branch name with `docs-`; admittedly that pragma is rather hard to find in the dev docs: https://docs.securedrop.org/en/latest/development/documentation_guidelines.html#testing-documentation-changes

No need to resubmit, just mentioning it in case it saves you time on a future contribution. :slightly_smiling_face: We'll kick CI to get the updated logic running for you, hopefully getting to green across the board.

**Comment by cji** [comment]
@conorsch 	oh cool! I will keep that in mind for next time. Thank you!

File: docs/configure_admin_workstation_post_install.rst [hunk 1]
```diff
@@ -63,3 +63,8 @@ of the authenticated hidden services, restart Tails and make
 sure to enable persistence.
 
 .. _Authenticated Tor Hidden Services: https://www.torproject.org/docs/tor-manual.html.en#HiddenServiceAuthorizeClient
+
+Back Up The Workstations
+------------------------------------
+
+USB drives can wear out, get lost, or otherwise become corrupted, making it very important to be sure to keep current backups. Follow the :ref:`Backup the Workstations <backup_workstations>` document to create a backup of your Secure Viewing Station, Admin Workstation, and Journalist Workstations after you've completed the installation and post-installation steps.
```
Comment (eloquence) [inline docs/configure_admin_workstation_post_install.rst]: Nit: We italicize SecureDrop-specific terminology (see https://docs.securedrop.org/en/release-0.12.1/development/documentation_guidelines.html and https://docs.securedrop.org/en/release-0.12.1/terminology.html ).

File: docs/configure_admin_workstation_post_install.rst [hunk 2]
```diff
@@ -63,3 +63,8 @@ of the authenticated hidden services, restart Tails and make
 sure to enable persistence.
 
 .. _Authenticated Tor Hidden Services: https://www.torproject.org/docs/tor-manual.html.en#HiddenServiceAuthorizeClient
+
+Back Up The Workstations
```
Comment (eloquence) [inline docs/configure_admin_workstation_post_install.rst]: Nit: The -> the (title case per https://docs.securedrop.org/en/release-0.12.1/development/documentation_guidelines.html although we're admittedly not super-consistent about this).

**Comment by eloquence** [comment]
LGTM, couple of nits noted inline if you have time, if not, we can fix in follow-up. :)

**Comment by cji** [comment]
@eloquence Thanks for the review (and for pointing out the style doc, I embarrassingly missed it 😅). I think I addressed your latest comments. Please let me know if I missed anything!

**Comment by conorsch** [comment]
Lovely. Thank you for the contribution, and for your patience while we got it in, @cji!

**Comment by cji** [comment]
@conorsch no problem, my pleasure! Thanks for all the feedback!

**Pull request merged**